contents/docs/user-guides/experimentation.mdx

```diff
@@ -0,0 +1,157 @@
+
+To ensure your experiment does not introduce any regressions, you can also track secondary metrics. This is particularly useful if you're optimizing a metric that has a natural trade-off with another metric (e.g. you can maximize conversion by reducing your price but then your total revenue may be reduced).
+
+Once your experiment is terminated, you can easily archive and share your results with other team members.
+
+### Results significance
+
+In order for your results and conclusions to be valid, any experiment must have a significant exposure. For instance, if you test a product change and only one user sees the change, you can't extrapolate from that single user that the change will be beneficial/detrimental for your entire user base. This is true for any experiment that uses the scientific method (e.g. this is also done when testing new drugs or vaccines).
+
+PostHog computes this significance for you automatically - we will let you know if your experiment has reached significant results or not. Once your experiment reaches significant results, it's safe to use those results to reach a conclusion and terminate the experiment. You can read more about how we do this in our 'Advanced' section below
+
+### Experimentation vs. Feature Flags
+
+Experimentation and Feature Flags are different features because they serve different purposes. **Experimentation** is used to test changes to your product where you want to maximize a specific metric (eg. testing a new flow to maximize conversion rate). **Feature Flags** are used for phased roll-outs (e.g. a new experimental feature you want to make sure works correctly) or as a way to control feature access. Read our [Feature Flags user guide](/docs/user-guides/feature-flags) for more information on this.
+
+Experimentation uses Feature Flags under the hood to handle user allocation to an experiment. You'll notice this when checking if a user is assigned to an experiment.
+
+## Using Experimentation
+
+### Creating an experiment
+
+Guide on how to create an experiment [here](https://docs.google.com/document/d/1DJmSY6ILZFmHhmRLTQ0L2oXGgFZCvGM4DF-zh92DIE8/edit?usp=sharing).
+
+### Running an experiment
+
+TODO: Guide on how to run an experiment here.
+
+### Terminating an experiment
+
+You decide when to terminate an experiment. A banner in your experiment results page will let you know if it's safe to terminate an experiment or not (regardless of the original estimated running time). If you terminate the experiment before you see this banner, results will not be significant (see Advanced for more details on this) and any conclusions could be completely wrong.
+
+Once you have decided to terminate an experiment there are a few things to do:
+
+-   [ ] Click on "Terminate the experiment" on the experiment page. This will ensure final results are kept.
+-   [ ] We recommend you roll-out the winning variant to all your users. TODO: Explain here how to do this with PostHog.
```

Comment:
I liked @clarkus idea of putting a button on the experiment finished page to "rollout to everyone". Alternatively, could link to the FF page to adjust yourself.

Reply:
Example of this in figma at https://www.figma.com/file/gQBj9YnNgD8YW4nBwCVLZf/PostHog-App?node-id=6679%3A32112